qiskit/providers/ibmq/job/ibmqjob.py

```diff
@@ -381,10 +386,10 @@ def submit(self) -> None:
             The job has started.
 
         Raises:
-            JobError: If an error occurred during job submit.
+            IBMQJobInvalidStateError: If an error occurred during job submit.
```

Comment:
Can you add the `IBMQJobApiError` to the docstring and be more specific about which one is raised for what?